fetch.js

```diff
@@ -292,6 +292,10 @@
       throw new RangeError('Invalid status code')
     }
 
+    var link = document.createElement('a')
+    link.href = url
```

Comment:
Is there a better way to do this? Would there be downsides to intializing `A` elements?